pipeline/tasks/gapic_tasks.py

```diff
@@ -61,10 +61,7 @@ def _move_to(self, gapic_config_dir, gapic_api_yaml):
         elif len(gapic_api_yaml) == 0:
             raise ValueError(error_fmt + 'No location specified')
         conf_out = os.path.abspath(gapic_api_yaml[0])
-        if os.path.exists(conf_out):
-            raise ValueError(error_fmt + 'File already exists')
```

Comment:
@garrettjonesgoogle Was condition #3 satisfied when you were doing that?

@geigerj That could also be argued as a command naming issue. I used to alias `git commit` as `git ci`, and `git checkout` as `git co`. Then I realized that running `git co .` in a sandbox full of unstaged changes when I wanted to run `git ci .` could really ruin my day. I changed the checkout alias to `git cho`, which is different enough that I'm no longer worried about it.